pillar/redash.sls

```diff
@@ -6,7 +20,34 @@ apache:
       servername: redash.open-contracting.org
       context:
         proxypass: http://localhost:9090/
+
+postgres:
+  version: 13
+  # Redash connections to Postgres are not sent over the localhost connection.
+  # Postgres access is locked down further in the Linode firewall.
+  public_access: true
+  configuration: redash
+  storage: ssd
+  type: oltp
+  nr_hugepages: 128
+  databases:
+    redash_db:
```

Comment:
FYI, instead of having every database be named `X_db`, `Y_db`, etc., we just name database `X`, `Y`, etc.